clients/drcachesim/tools/view.cpp

```diff
@@ -424,9 +424,13 @@ view_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             std::cerr << "<marker: system call trace end>\n";
             break;
         case TRACE_MARKER_TYPE_BRANCH_TARGET:
+            // These are not expected to be visible but we handle nonetheless.
             std::cerr << "<marker: indirect branch target 0x" << std::hex
                       << memref.marker.marker_value << std::dec << ">\n";
             break;
+        case TRACE_MARKER_TYPE_WAIT:
+            std::cerr << "<marker: wait for another core>\n";
```

Comment:
We should add more docx on how the user should interpret this. So the current core was waiting, how long?

Reply:
"  A new marker is emitted each
     * time the tool analysis framework requests a new record from the scheduler and
     * is given a wait status.  There are no units of time here but each repetition
     * is roughly the time where a regular record could have been read and passed
     * along."